test/Microsoft.SqlTools.ServiceLayer.TestDriver/Tests/QueryExecutionTests.cs

```diff
@@ -323,5 +324,38 @@ public class QueryExecutionTests
                 await testHelper.Disconnect(queryTempFile.FilePath);
             }
         }
+
+        [Fact]
+        public async Task NoOpQueryReturnsMessage()
```

Comment:
Shouldn't we also have a test that covers the error message case? I don't see one of these in this test file